scripts/run_acceptance_tests.sh

```diff
@@ -12,6 +12,6 @@ export BOSH_CLIENT_SECRET
 export BOSH_URL
 export BBR_BUILD_PATH
 
-go get github.com/onsi/ginkgo/ginkgo
-glide install --strip-vendor
+#go get github.com/onsi/ginkgo/ginkgo
```

Comment:
We had to comment these out after our initial `glide install --strip-vendor` because ginkgo kept running all of the tests in the `vendor` directory. We don't think you'll want to check this in, but we wanted to know what we were doing wrong. Thanks :)

Reply:
We think this might be related to using an older version of ginkgo. We have never seen this problem in our own pipeline or local machine, so we won't accept this change for now. If you are using an older version of ginkgo, could you please try to bump ginkgo and let us know if the problem still exists?